violet-framework/src/main/java/com/horstmann/violet/workspace/editorpart/behavior/AddNodeBehavior.java

```diff
@@ -91,15 +91,16 @@ public void onMouseClicked(MouseEvent event)
 
     private boolean checkIfNumberOfInitialNodesIsMoreThanOne(INode newNode)
     {
+        boolean result = false;
         if (newNode instanceof IInitialStateNode)
         {
             int numberOfInitialNodes = getNumberOfInitialNodes();
```

Comment:
Meybe you didn't notice but i make this variable because of your wish. If you check version before i didn't have it and just simple return true or false. You said that i should change it to don't generate double return in method and i agreed with that.